styles/Vocab/Plone/reject.txt

```diff
@@ -2,4 +2,8 @@ node
 nodejs
 javascript
 js
-Javascript
\ No newline at end of file
+Javascript
+volto
+plone
+nodejs
+razzle 
```

Comment:
Before adding these entries, did Vale report them as misspellings?

After adding them, did Vale report that their usage in the docs should be corrected?

if so, then did you correct their usage in the docs?

Reply:
I'm curious to see the result of Vale's output at each change, showing the counts of warnings, errors, and suggestions, before and after. After corrections are made in the docs, does the configuration change make things worse or improve them?